### PR TITLE
Copied v1.12 files over from eeglab plugin download

### PR DESCRIPTION
**Pull request by cboulay**
and modified to suit new structure.

It seems as though git diff thinks much has changed between the files but obviously this is not true. Maybe eol mismatch? Should I handle it or just leave it?

### CODE REVIEW

**Comment by dmedine** [comment]
Isn't there some eol magic via git? I always forget what it is. But, as 
far as I know 1.12 is pretty close to 1.11. I think it's okay to leave 
it for now. I will take a look at it to make sure it is ok. I'm pretty 
sure it's fine, though.

Thanks!

**Comment by dmedine** [comment]
I'll check it out an pull it this afternoon.

**Comment by dmedine** [comment]
Did not have time this afternoon. Tomorrow.

**Comment by chkothe** [comment]
We could also merge the changes in by hand using a merge tool that can
ignore line endings and/or whitespace as a workaround, and then commit that.

**Comment by cboulay** [comment]
I sorted out the line endings on my end and refreshed the pull request.

With some other repositories that I've worked on, it was considered better to force push a pull request than to delete it, delete the branch, make a new branch, and make a new pull request. I hope you don't mind that's what I did here. If you've already checked out this branch then you'll have to force pull.

**Comment by chkothe** [comment]
Ok, great! Yes, I don't have an issue with folks force-pushing to rewrite
their PRs (unless others have actively contributed to it already, which
almost never happens).

File: Matlab/xdf/load_xdf.m [hunk 1]
```diff
@@ -502,7 +503,7 @@
                 segments(r).t_begin = temp(k).time_stamps(range(1));
                 segments(r).t_end = temp(k).time_stamps(range(2));
                 segments(r).duration = segments(r).t_end - segments(r).t_begin;
-                segments(r).effective_srate = (segments(r).num_samples-1)/ segments(r).duration;
```
Comment (chkothe) [inline Matlab/xdf/load_xdf.m]: This change here was applied to 1.11 in a 3rd party contribution without bumping up the version number, and was unfortunately not reflected in the 1.12 version. It's probably best to carry this line over into 1.12 in the repo.

**Comment by cboulay** [comment]
OK. If that's all then please go ahead and merge then attempt to make_release.py
tags and releases are easily deleted so there's no real problem there.

And if you forget to add the mex files to the Matlab/xdf folder before creating the release, they can always be added to the release manually via the web interface.

**Pull request merged**